platform/iphone/os_iphone.cpp

```diff
@@ -30,11 +30,12 @@
 
 #include "os_iphone.h"
 
-#include "drivers/gles2/rasterizer_gles2.h"
+//#include "drivers/gles2/rasterizer_gles2.h"
```

Comment:
@akien-mga , you mean just remove the line all together since gles2 will soon be trashed anyway? Just being overly careful here :)

@bojidar-bg not probably, it has:) the gles3 implementation doesn't seem to have that much in common with the gles2 on anymore ;)